config/initializers/new_framework_defaults_6_0.rb

```diff
@@ -0,0 +1,45 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Same question as with the front office – are we still using this post-update?

Reply:
Removed it.